principles.md

```diff
@@ -3,12 +3,16 @@
 <br>
 
 ### Proposal Principles to Use to Achieve the Goal​
-- Boy Scout Rule​
+- Make it work, make it right, make it fast - Kent Beck
+  - Make it work: the beaufitul code means nothing to users if it doesn't work, which mean the working code should meet the minimum requirements for the business to call project a success. 
+  - Make it right: 1 written working code will be read at least 10 times and these code is going to be changed as the nature of the software development. Therefore, Add bells and whistles to make the program less prone to error and more feature rich.
+  - Make it fast: Find and eliminate waste in the process. Some assumtions from the start will have been incorrect. Remove unecessary business logic. Included in this step is to improve code for better performance.
+- ~~Boy Scout Rule​~~ - this is minor. A lot of time it's better don't changes that is working.
```

Comment:
We shouldn't change the perfect code that working as expected :) Unfortunately most of the time, it is unrealistic.
That's why, we would need to improve the code constantly to make it better.
I don't think this is minor. Remember, do small things consistently add up to big things.